gatsby-node.js

```diff
@@ -83,6 +83,7 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
             iconName
             originalFilePath
             productStub
+            indexMode
```

Comment:
I think `indexCards` might be a better name for this, unless we anticipate latching on more behavior to this value. Otherwise it's probably nicer to have the flexibility